components/tabs/tabs.js

```diff
@@ -0,0 +1,388 @@
+import '../colors/colors.js';
+import '../icons/icon.js';
+import '../../helpers/queueMicrotask.js';
+import './tab.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { getNextFocusable, getPreviousFocusable } from '../../helpers/focus.js';
+import { ArrowKeysMixin } from '../../mixins/arrow-keys-mixin.js';
+import { bodyCompactStyles } from '../typography/styles.js';
+import { findComposedAncestor } from '../../helpers/dom.js';
+import { LocalizeStaticMixin } from '../../mixins/localize-static-mixin.js';
+import {repeat} from 'lit-html/directives/repeat';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+//const reduceMotion = matchMedia('(prefers-reduced-motion: reduce)').matches;
```

Comment:
This will be used if necessary in the code where normally we would wait for animation/transition events.  If reduced motion is preferred and we turn off transitions, that code will need to conditionally account for the absence of those events.